### ISSUE TITLE
Is there functionality that shows the latest apps, flows, connectors that were created in the last day?

### ISSUE THREAD

**Issue by PPCan** [microsoft/coe-starter-kit#9319]
### Does this question already exist in our backlog?

- [x] I have checked and confirm this is a new question.

### What is your question?

There used to be a flow template that did this, that wasn't part of the COE Toolkit but the template is gone.

We would like to know what apps, flows or connectors have been created, so we can review and contact the owner if needed.

### What solution are you experiencing the issue with?

None

### What method are you using to get inventory and telemetry?

None

**Comment by Jenefer-Monroe** [comment]
Hello, we do not have that functionality today, no. But happy to add to the backlog. 
Do note that we have competing priorities so this is not something we will be able to get to soon.

**Comment by PPCan** [comment]
Yes please do add it.  It is very helpful to know what has been added to the tenant in general, especially personal prod.

This topic is critical to good governance, but isn't addressed anywhere.

There is the 'welcome' email for new makers, however that isn't as useful as an email that says 'we know you've made something, please go here and tell us what it is' with backend functionality.

I know the audit process/compliance flows do something similar, however they are too complex really.  A simple, 'here's what's new', so we can communicate with the maker would be easier to implement, and the compliance could be a +1 to that.

Hope that makes sense.

**Comment by Ayesha-320** [comment]
Hi @PPCan,
Thanks for raising this governance-focused request. While the CoE Starter Kit doesn’t currently support tracking newly created apps, flows, or connectors within the last 24 hours, we’ve added this to the backlog. A proposed enhancement is to extend the existing Admin | Sync Template v3 (Driver) flow to filter inventory data by creation timestamp and notify admins. Tagging @RajeevPentyala and @MohammadRizwan for visibility. Closing this issue for now
